### ISSUE TITLE
Error when uploading image to ticket task

### ISSUE THREAD

**Issue by Tbence95** [glpi-project/glpi#15885]
### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### Is there an existing issue for this?

- [X] I have searched the existing issues

### Version

10.0.10

### Bug description

I try to attach jpg image (~4MB) to a ticket task with drag and drop and it takes very long time, and after the task creation there will be an empty task (without picture and text).
I tried this process with a smaller image (200KB) and it works.
I tried to upload the 4MB picture with "Choose files" and browse it from my PC. --It works.
So it works with browse option, but with drag&drop doesnt.

I enabled debug mode in GLPI and there is an error message at tasks with the 4MB picture.
The error message is:
PHP Deprecated function (8192): preg_replace(): Passing null to parameter #3 ($subject) of type array|string is deprecated in /web/glpi/src/RichText/RichText.php at line 352

OS: CentOS 7
Web server: Apache 2.4.6
PHP: 8.1.24
SQL: MariaDB 10.5.22

Anyone has a suggestion what do we have to do, or is it a bug?
Thanks in advance

### Relevant log output

```shell
PHP Deprecated function (8192): preg_replace(): Passing null to parameter #3 ($subject) of type array|string is deprecated in /web/glpi/src/RichText/RichText.php at line 352
```

### Steps To reproduce

With the environment I have wroted above, try to drag&drop an image to a ticket task with size ~4MB or above.

**Comment by Tbence95** [comment]
Any update on this?

**Comment by cedric-anne** [comment]
Could you please provide the full error trace? You could find it in `files/_log/php-errors.log`.

**Comment by Tbence95** [comment]
The files/_log/php-errors.log is now empty, but the image upload doesn't work.
Whetn I created the issue, the only error was the line, that I mentioned above.

**Comment by cedric-anne** [comment]
When an error is present in the output, it contains only the error summary, but the full trace is always added into the `files/_log/php-errors.log` file. Without this full trace, it makes investigation a pain.

If you still reproduce the issue, and if this error is still present in output, then its full trace should be present in the log file.

**Comment by Tbence95** [comment]
I enabled debug mode, i send the content of the log file.

``[2023-11-08 08:00:02] glpiphplog.WARNING: Test logger
[2023-11-08 08:01:01] glpiphplog.WARNING: Test logger
2023-11-08 10:02:32 [15@glpi.domain.com]
includeLocales(statusertask_report_title, reports) => not found
2023-11-08 10:02:33 [15@glpi.domain.com]
includeLocales(statusertask_report_title, reports) => not found
2023-11-08 10:02:33 [15@glpi.domain.com]
includeLocales(statusertask_report_title, reports) => not found
2023-11-08 10:02:33 [15@glpi.domain.com]
includeLocales(statusertask_report_title, reports) => not found
2023-11-08 10:02:33 [15@glpi.domain.com]
includeLocales(statusertask_report_title, reports) => not found
2023-11-08 10:02:33 [15@glpi.domain.com]
includeLocales(statusertask_report_title, reports) => not found
2023-11-08 10:02:33 [15@glpi.domain.com]
includeLocales(statusertask_report_title, reports) => not found
2023-11-08 10:02:34 [15@glpi.domain.com]
includeLocales(statusertask_report_title, reports) => not found
2023-11-08 10:02:34 [15@glpi.domain.com]
includeLocales(statusertask_report_title, reports) => not found
2023-11-08 10:02:34 [15@glpi.domain.com]
includeLocales(statusertask_report_title, reports) => not found
2023-11-08 10:02:34 [15@glpi.domain.com]
includeLocales(statusertask_report_title, reports) => not found
2023-11-08 10:02:34 [15@glpi.domain.com]
includeLocales(statusertask_report_title, reports) => not found
2023-11-08 10:02:34 [15@glpi.domain.com]
includeLocales(statusertask_report_title, reports) => not found
2023-11-08 10:02:34 [15@glpi.domain.com]
includeLocales(statusertask_report_title, reports) => not found
2023-11-08 10:02:34 [15@glpi.domain.com]
includeLocales(statusertask_report_title, reports) => not found
2023-11-08 10:02:34 [15@glpi.domain.com]
includeLocales(statusertask_report_title, reports) => not found
2023-11-08 10:02:34 [15@glpi.domain.com]
includeLocales(statusertask_report_title, reports) => not found
2023-11-08 10:02:34 [15@glpi.domain.com]
includeLocales(statusertask_report_title, reports) => not found
2023-11-08 10:02:34 [15@glpi.domain.com]
includeLocales(statusertask_report_title, reports) => not found
2023-11-08 10:02:34 [15@glpi.domain.com]
includeLocales(statusertask_report_title, reports) => not found
2023-11-08 10:02:34 [15@glpi.domain.com]
includeLocales(statusertask_report_title, reports) => not found
2023-11-08 10:02:34 [15@glpi.domain.com]
includeLocales(statusertask_report_title, reports) => not found
2023-11-08 10:02:34 [15@glpi.domain.com]
includeLocales(statusertask_report_title, reports) => not found
2023-11-08 10:02:34 [15@glpi.domain.com]
includeLocales(statusertask_report_title, reports) => not found
2023-11-08 10:02:34 [15@glpi.domain.com]
includeLocales(statusertask_report_title, reports) => not found
2023-11-08 10:02:34 [15@glpi.domain.com]
includeLocales(statusertask_report_title, reports) => not found
2023-11-08 10:02:34 [15@glpi.domain.com]
includeLocales(statusertask_report_title, reports) => not found
2023-11-08 10:02:34 [15@glpi.domain.com]
includeLocales(statusertask_report_title, reports) => not found
2023-11-08 10:02:34 [15@glpi.domain.com]
includeLocales(statusertask_report_title, reports) => not found
2023-11-08 10:02:34 [15@glpi.domain.com]
includeLocales(statusertask_report_title, reports) => not found
2023-11-08 10:02:34 [15@glpi.domain.com]
includeLocales(statusertask_report_title, reports) => not found
2023-11-08 10:02:34 [15@glpi.domain.com]
includeLocales(statusertask_report_title, reports) => not found
2023-11-08 10:02:35 [15@glpi.domain.com]
includeLocales(statusertask_report_title, reports) => not found
2023-11-08 10:02:35 [15@glpi.domain.com]
includeLocales(statusertask_report_title, reports) => not found
2023-11-08 10:02:35 [15@glpi.domain.com]
includeLocales(statusertask_report_title, reports) => not found
2023-11-08 10:02:35 [15@glpi.domain.com]
includeLocales(statusertask_report_title, reports) => not found
2023-11-08 10:02:35 [15@glpi.domain.com]
includeLocales(statusertask_report_title, reports) => not found
2023-11-08 10:02:35 [15@glpi.domain.com]
includeLocales(statusertask_report_title, reports) => not found
2023-11-08 10:02:35 [15@glpi.domain.com]
includeLocales(statusertask_report_title, reports) => not found
2023-11-08 10:02:35 [15@glpi.domain.com]
includeLocales(statusertask_report_title, reports) => not found
2023-11-08 10:02:35 [15@glpi.domain.com]
includeLocales(statusertask_report_title, reports) => not found
[2023-11-08 10:02:36] glpiphplog.NOTICE:   *** PHP Deprecated function (8192): preg_replace(): Passing null to parameter #3 ($subject) of type array|string is deprecated in /web/glpi/src/RichText/RichText.php at line 352
  Backtrace :
  src/RichText/RichText.php:352                      preg_replace()
  src/RichText/RichText.php:271                      Glpi\RichText\RichText::fixImagesPath()
  src/RichText/RichText.php:113                      Glpi\RichText\RichText::normalizeHtmlContent()
  ...ion/View/Extension/DataHelpersExtension.php:192 Glpi\RichText\RichText::getTextFromHtml()
  ...ates/b5/b540183b1332ad4a177eb7ba8bcaf20f.php:76 Glpi\Application\View\Extension\DataHelpersExtension->getTextFromHtml()
  vendor/twig/twig/src/Template.php:171              __TwigTemplate_8b1685cd1b2cfaa332548ea782232dff->block_timeline_card()
  ...ates/a4/a4a2db6ed5602e71abd50e76ae8800be.php:48 Twig\Template->displayBlock()
  vendor/twig/twig/src/Template.php:394              __TwigTemplate_4bff84a61e9b886d83310ebf90c196f0->doDisplay()
  vendor/twig/twig/src/Template.php:367              Twig\Template->displayWithErrorHandling()
  ...ates/b5/b540183b1332ad4a177eb7ba8bcaf20f.php:59 Twig\Template->display()
  vendor/twig/twig/src/Template.php:394              __TwigTemplate_8b1685cd1b2cfaa332548ea782232dff->doDisplay()
  vendor/twig/twig/src/Template.php:367              Twig\Template->displayWithErrorHandling()
  vendor/twig/twig/src/Template.php:379              Twig\Template->display()
  vendor/twig/twig/src/TemplateWrapper.php:40        Twig\Template->render()
  .../twig/twig/src/Extension/CoreExtension.php:1349 Twig\TemplateWrapper->render()
  ...tes/ec/ec6cbb8a2efe766fe2b94b1b85106708.php:420 twig_include()
  vendor/twig/twig/src/Template.php:394              __TwigTemplate_cf9582a178c9aed3fd76ab8a0bf60b2b->doDisplay()
  vendor/twig/twig/src/Template.php:367              Twig\Template->displayWithErrorHandling()
  vendor/twig/twig/src/Template.php:379              Twig\Template->display()
  vendor/twig/twig/src/TemplateWrapper.php:40        Twig\Template->render()
  .../twig/twig/src/Extension/CoreExtension.php:1349 Twig\TemplateWrapper->render()
  ...tes/24/24ef781d212992b008f998231b80b473.php:131 twig_include()
  vendor/twig/twig/src/Template.php:394              __TwigTemplate_c861bb866645fae902b77b643975f9f3->doDisplay()
  vendor/twig/twig/src/Template.php:367              Twig\Template->displayWithErrorHandling()
  vendor/twig/twig/src/TemplateWrapper.php:47        Twig\Template->display()
  src/Application/View/TemplateRenderer.php:184      Twig\TemplateWrapper->display()
  src/Ticket.php:4409                                Glpi\Application\View\TemplateRenderer->display()
  src/CommonGLPI.php:677                             Ticket->showForm()
  ajax/common.tabs.php:116                           CommonGLPI::displayStandardTab()
  public/index.php:82                                require()

[2023-11-08 10:02:36] glpiphplog.NOTICE:   *** PHP Deprecated function (8192): preg_replace(): Passing null to parameter #3 ($subject) of type array|string is deprecated in /web/glpi/src/RichText/RichText.php at line 352
  Backtrace :
  src/RichText/RichText.php:352                      preg_replace()
  src/RichText/RichText.php:271                      Glpi\RichText\RichText::fixImagesPath()
  src/RichText/RichText.php:63                       Glpi\RichText\RichText::normalizeHtmlContent()
  src/RichText/RichText.php:299                      Glpi\RichText\RichText::getSafeHtml()
  ...ion/View/Extension/DataHelpersExtension.php:230 Glpi\RichText\RichText::getEnhancedHtml()
  ...ates/b5/b540183b1332ad4a177eb7ba8bcaf20f.php:80 Glpi\Application\View\Extension\DataHelpersExtension->getEnhancedHtml()
  vendor/twig/twig/src/Template.php:171              __TwigTemplate_8b1685cd1b2cfaa332548ea782232dff->block_timeline_card()
  ...ates/a4/a4a2db6ed5602e71abd50e76ae8800be.php:48 Twig\Template->displayBlock()
  vendor/twig/twig/src/Template.php:394              __TwigTemplate_4bff84a61e9b886d83310ebf90c196f0->doDisplay()
  vendor/twig/twig/src/Template.php:367              Twig\Template->displayWithErrorHandling()
  ...ates/b5/b540183b1332ad4a177eb7ba8bcaf20f.php:59 Twig\Template->display()
  vendor/twig/twig/src/Template.php:394              __TwigTemplate_8b1685cd1b2cfaa332548ea782232dff->doDisplay()
  vendor/twig/twig/src/Template.php:367              Twig\Template->displayWithErrorHandling()
  vendor/twig/twig/src/Template.php:379              Twig\Template->display()
  vendor/twig/twig/src/TemplateWrapper.php:40        Twig\Template->render()
  .../twig/twig/src/Extension/CoreExtension.php:1349 Twig\TemplateWrapper->render()
  ...tes/ec/ec6cbb8a2efe766fe2b94b1b85106708.php:420 twig_include()
  vendor/twig/twig/src/Template.php:394              __TwigTemplate_cf9582a178c9aed3fd76ab8a0bf60b2b->doDisplay()
  vendor/twig/twig/src/Template.php:367              Twig\Template->displayWithErrorHandling()
  vendor/twig/twig/src/Template.php:379              Twig\Template->display()
  vendor/twig/twig/src/TemplateWrapper.php:40        Twig\Template->render()
  .../twig/twig/src/Extension/CoreExtension.php:1349 Twig\TemplateWrapper->render()
  ...tes/24/24ef781d212992b008f998231b80b473.php:131 twig_include()
  vendor/twig/twig/src/Template.php:394              __TwigTemplate_c861bb866645fae902b77b643975f9f3->doDisplay()
  vendor/twig/twig/src/Template.php:367              Twig\Template->displayWithErrorHandling()
  vendor/twig/twig/src/TemplateWrapper.php:47        Twig\Template->display()
  src/Application/View/TemplateRenderer.php:184      Twig\TemplateWrapper->display()
  src/Ticket.php:4409                                Glpi\Application\View\TemplateRenderer->display()
  src/CommonGLPI.php:677                             Ticket->showForm()
  ajax/common.tabs.php:116                           CommonGLPI::displayStandardTab()
  public/index.php:82                                require()

2023-11-08 10:02:36 [15@glpi.domain.com]
includeLocales(statusertask_report_title, reports) => not found
2023-11-08 10:02:36 [15@glpi.domain.com]
includeLocales(statusertask_report_title, reports) => not found
2023-11-08 10:02:36 [15@glpi.domain.com]
includeLocales(statusertask_report_title, reports) => not found
2023-11-08 10:02:36 [15@glpi.domain.com]
includeLocales(statusertask_report_title, reports) => not found
2023-11-08 10:02:37 [15@glpi.domain.com]
includeLocales(statusertask_report_title, reports) => not found
2023-11-08 10:02:37 [15@glpi.domain.com]
includeLocales(statusertask_report_title, reports) => not found
2023-11-08 10:02:37 [15@glpi.domain.com]
includeLocales(statusertask_report_title, reports) => not found
2023-11-08 10:02:37 [15@glpi.domain.com]
includeLocales(statusertask_report_title, reports) => not found
2023-11-08 10:02:37 [15@glpi.domain.com]
includeLocales(statusertask_report_title, reports) => not found
2023-11-08 10:02:37 [15@glpi.domain.com]
includeLocales(statusertask_report_title, reports) => not found
2023-11-08 10:02:37 [15@glpi.domain.com]
includeLocales(statusertask_report_title, reports) => not found
2023-11-08 10:02:37 [15@glpi.domain.com]
includeLocales(statusertask_report_title, reports) => not found
2023-11-08 10:02:37 [15@glpi.domain.com]
includeLocales(statusertask_report_title, reports) => not found
2023-11-08 10:02:38 [15@glpi.domain.com]
includeLocales(statusertask_report_title, reports) => not found
2023-11-08 10:02:38 [15@glpi.domain.com]
includeLocales(statusertask_report_title, reports) => not found
2023-11-08 10:02:38 [15@glpi.domain.com]
includeLocales(statusertask_report_title, reports) => not found
2023-11-08 10:03:19 [15@glpi.domain.com]
includeLocales(statusertask_report_title, reports) => not found

**Comment by RobertoMarcelino** [comment]
Same problem with the ticket solution.

![image](https://github.com/glpi-project/glpi/assets/57845086/99469b9b-906b-4cc3-9ddf-79ddd7b0390c)

**Comment by Tbence95** [comment]
Any updates on this?

**Comment by RobertoMarcelino** [comment]
This problem was fixed in version 10.0.11.
